testsuite/integration/web/src/test/java/org/jboss/as/test/integration/web/security/form/WebSecuritySimpleRoleMappingTestCase.java

```diff
@@ -43,26 +43,19 @@
 @RunAsClient
 @ServerSetup(WebSimpleRoleMappingSecurityDomainSetup.class)
 @Category(CommonCriteria.class)
-public class WebSecurityJBossSimpleRoleMappingTestCase extends AbstractWebSecurityFORMTestCase {
+public class WebSecuritySimpleRoleMappingTestCase extends AbstractWebSecurityFORMTestCase {
 
     @Deployment(testable = false)
     public static WebArchive deployment() throws Exception {
-
-        WebArchive war = ShrinkWrap.create(WebArchive.class, "web-secure.war");
-        war.addClasses(SecuredServlet.class);
-
-        war.addAsWebResource(WebSecurityJBossSimpleRoleMappingTestCase.class.getPackage(), "login.jsp", "login.jsp");
-        war.addAsWebResource(WebSecurityJBossSimpleRoleMappingTestCase.class.getPackage(), "error.jsp", "error.jsp");
-
-        war.addAsWebInfResource(WebSecurityJBossSimpleRoleMappingTestCase.class.getPackage(), "jboss-web-role-mapping.xml",
-                "jboss-web.xml");
-        war.addAsWebInfResource(WebSecurityJBossSimpleRoleMappingTestCase.class.getPackage(), "web.xml", "web.xml");
-
-        war.addAsResource(WebSecurityJBossSimpleRoleMappingTestCase.class.getPackage(), "users.properties", "users.properties");
-        war.addAsResource(WebSecurityJBossSimpleRoleMappingTestCase.class.getPackage(), "roles.properties", "roles.properties");
+        WebArchive war = prepareDeployment("jboss-web.xml");
         return war;
     }
 
+    @BeforeClass
+    public static void beforeClass() {
+        AssumeTestGroupUtil.assumeElytronProfileTestsEnabled();
```

Comment:
I prefer to keep it executed even when elytron profile is used. Use the `AssumeTestGroupUtil` just for failing tests - following the contract which says:

> Assume for test failures when running with Elytron profile enabled.